frontend/src/Components/MentorRole.js

```diff
@@ -0,0 +1,116 @@
+import React, { useState } from "react";
```

Comment:
I also feel like, you dont need a separate component. Instead have a generic component and send in the value for the question to render based on the role selected